doc/Extensions/Galera-Cluster.md

```diff
@@ -0,0 +1,154 @@
+source: Extensions/Galera-Cluster.md
+path: blob/master/doc/
+
+# MariaDB Galera Cluster
+
+This is currently being tested, use at your own risk.
+
+LibreNMS can be used with a MariaDB Galera Cluster. This is a Multi Master cluster, meaning each
+node in the cluster can read and write to the database. They all have the same ability. LibreNMS will
+randomly choose a working node to read and write requests to. 
+
+
+For more information see
+<https://laravel.com/docs/8.x/database#read-and-write-connections>
```

Comment:
You can use this link to avoid specific version link:
`https://laravel.com/docs/database#read-and-write-connections`

Reply:
Fixed